google/datalab/contrib/mltoolbox/commands/_ml.py

```diff
@@ -49,6 +57,78 @@ def ml(line, cell=None):
 Use "%ml <command> -h" for help on a specific command.
 """)
 
+  analyze_parser = parser.subcommand(
```

Comment:
textwrap.dedent(""".... could be used to keep this block of text indented with the code, up to you.